### DISCUSSION TITLE
Snippets: in the replacement text if `p` pressed before any other letter, it acts a paste

### DISCUSSION THREAD

**Discussion by rjshrjndrn** [LazyVim/LazyVim#4901] [Q&A] [answered]
for example, in git commit you type `chore` and auto complete pops up
`chore(scope): title`
if I keep my cursor at `s` (of scope) and press p (say, i'm working on a plugin), then rather than p, it paste the buffer.

Anybody faced such issue?

**Comment by rjshrjndrn** [comment]
The issue was because that I had a mapping 
`keymap("v", "p", '"_dP', opts)`

But I want to keep that maaping, as it'll keep the visual paste sane. Is there any way to keep this, and solve the wierd paste in snippets?

**Reply by Kayzels** [reply, marked as answer]
The `"v"` there is for both visual and select modes, and the snippets will be in select mode. To have it only work for visual mode, change the `"v"` to `"x"`.

**Reply by rjshrjndrn** [reply]
Awesome.. everyday new thing in vim :1234:
